device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/pg_profile_lookup.ini

```diff
@@ -0,0 +1,17 @@
+# PG lossless profiles.
+# speed cable size    xon  xoff threshold xon_offset
+  10000  5m   4608    4608 35776   -3        4608
+  25000  5m   4608    4608 53248   -3        4608
+  40000  5m   4608    4608 66560   -3        4608
+  50000  5m   4608    4608 33792   -3        4608
+ 100000  5m   4608    4608 49408   -3        4608
+  10000  40m  4608    4608 37024   -3        4608
+  25000  40m  4608    4608 56160   -3        4608
+  40000  40m  4608    4608 71552   -3        4608
+  50000  40m  4608    4608 41234   -3        4608
+ 100000  40m  4608    4608 54528   -3        4608

```

Comment:
Why 100G PG headroom is smaller than 25G @40m

Reply:
@baiwei0427

Reply:
@vmittal-msft some values here do not make sense to me. let's go through these numbers offline.

Reply:
Discussed this with Vineet, he will be updating the correct numbers

Reply:
Updated the file with corrected values.